book/src/external-provisioner.md

```diff
@@ -10,6 +10,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-provisioner v3.1.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.1.0) | [release-3.1](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.1) | [v1.5.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | - | k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0 | v1.21 | - | v1.23
```

Comment:
The min version says 1.20 and recommended version is 1.22

Reply:
actually no change in min or recommended version, iic its just a miss, any way kept as 1.20 and 1.22